.prettierrc.js

```diff
@@ -3,12 +3,6 @@ module.exports = {
   singleQuote: true,
   trailingComma: 'es5',
   overrides: [
-    {
-      files: 'pnpm-lock.yaml',
-      options: {
-        rangeEnd: 0, // default: Infinity
-      },
-    },
```

Comment:
Add this back

Reply:
Yeah the merge of the revert commit introduced a few mistakes i'll fix it